guides/ansible_engine/4-runplaybook/README.md

```diff
@@ -0,0 +1,37 @@
+# Exercise 4 - Running the apache-basic-playbook
+
+Congratulations!  You just wrote a playbook that incorporates some key Ansible concepts that you use in most if not all of your future playbooks.  Before you get too excited though, we should probably make sure it actually runs.
+
+So, lets do that now.
+
+## Section 1 - Running your new apache playbook
+
+### Step 1:
+Make sure you are in the right directory and create a host file.
+
+```bash
+cd ~/apache-basic-playbook
+```
+
+---
+**NOTE**
+Since you already did the work of creating a host file in Lession 1.0, feel free to re-use it.
+---
+
+### Step 2:
+Run your playbook
+
+```bash
+ansible-playbook --private-key#~/.ssh/workshopname-tower -i ../hosts site.yml
```

Comment:
The examples should not be using --private-key or -i here. They should be in the inventory file or `ansible.conf`. The AWS provisioner does use a private key (rather it setups up a username account with password) and generates a configuration file for students that points to the generated inventory file. A case could be made for showing students how to create an `ansible.conf`, but seems a bit more advance for the time and scope of it's companion.

Reply:
This is a mixup of redhatgov.io vs. ansible.redhatgov.io  
This should not be there.  We need to reference ansible.redhatgov.io